tests/python/pants_test/backend/python/tasks/test_ctypes.py

```diff
@@ -0,0 +1,117 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Can bump to 2018

Reply:
Good catch. This might be something that a linter could help with in the future?